### PR TITLE
eclipse-temurin: Bump JDK11, JDK17 and JDK21

### PR DESCRIPTION
**Pull request by plebcity**
Bumped to 21.0.2, 17.0.10 and 11.0.22. 8u401 isn't available yet but we can merge this already so people can get the latest bug fixes, especially for JDK 21.

### CODE REVIEW

**Comment by gdams** [comment]
We haven't released all architectures yet so this isn't ready to merge just yet I'm afraid

**Comment by plebcity** [comment]
> We haven't released all architectures yet so this isn't ready to merge just yet I'm afraid

Where can I see the progress on this?

**Comment by gdams** [comment]
> > We haven't released all architectures yet so this isn't ready to merge just yet I'm afraid
> 
> Where can I see the progress on this?

https://github.com/adoptium/temurin/issues/17

**Comment by gdams** [comment]
LGTM, I'll follow up with a separate PR to update 8 and 17 when they are ready

**Comment by gdams** [comment]
> LGTM, I'll follow up with a separate PR to update 8 and 17 when they are ready

@plebcity JDK17 is ready now

File: library/eclipse-temurin [hunk 1]
```diff
@@ -338,89 +338,88 @@ Constraints: nanoserver-1809, windowsservercore-1809
 #------------------------------v21 images---------------------------------
 Tags: 21.0.1_12-jdk-alpine, 21-jdk-alpine, 21-alpine
```
Comment (tianon) [inline library/eclipse-temurin:339]: @gdams shouldn't this be updated to `21.0.2_13` as well?  (or is that one of the builds that's still pending?)

Comment (gdams) [reply library/eclipse-temurin:339]: that build is still pending, I think we're safe to merge without)

Comment (gdams) [reply library/eclipse-temurin:339]: Alpine for JDK21 is also good to go now. @plebcity can you get this updated please and include the JDK17 updates

Comment (plebcity) [reply library/eclipse-temurin:339]: I grabbed the output of the dockerhub_doc_config_update.sh file and replaced the eclipse-temurin file with the output. So it should all be updated now.

Comment (gdams) [reply library/eclipse-temurin:339]: @plebcity run it again, we've published JDK8u now

**Review by gdams — COMMENTED** [review]
LGTM - this is now ready for merge

**Review by tianon — APPROVED** [review]

**Pull request merged**